src/ORM/Marshaller.php

```diff
@@ -289,6 +294,7 @@ public function many(array $data, array $options = [])
     protected function _belongsToMany(Association $assoc, array $data, $options = [])
     {
         $associated = isset($options['associated']) ? $options['associated'] : [];
+        $forceTargetSave = isset($options['forceTargetSave']) ? $options['forceTargetSave'] : false;
```

Comment:
The option makes no indication that is it is specific to belongsToMany associations either. Having this option no-op on all the other association types would be surprising to me. Having inconsistent behavior is not ideal.